featuretools/entityset/deserialize.py

```diff
@@ -59,6 +67,19 @@ def description_to_entityset(description, **kwargs):
     Returns:
         entityset (EntitySet) : Instance of :class:`.EntitySet`.
     '''
+    current = SCHEMA_VERSION.split('.')
+    saved = description.get('schema_version').split('.')
+    error_text = ('Unable to load features. The schema version of the saved '
```

Comment:
This error message is wrong. It says `features`. It should say `Unable to load entityset`.
Also, if we are going to copy a method from the feature list serialization check, we should make a modular function, and call that. That way there is no copied code.